docs/components/slider.md

```diff
@@ -0,0 +1,9 @@
+# Slider
+
+<DemoContainer>
+  <Slider :min="1000" :max="10000" :step="1000"/>
+</DemoContainer>
+
+```vue
+<Slider :min="1000" :max="10000" :step="1000"/>
+```
```

Comment:
For some reason I can only add an extra 0 in the textbox (can't edit it in any other way)